### PR TITLE
{bio}[intel/2017b] eXpress v1.5.1, protobuf v3.5.1

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

edit: depends on ~~#5736~~

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2407.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/9ff99ebd000070181de911d43b5b7337 for a full test report.

**Comment by verdurin** [comment]
@boegel set a build of this going last night and it failed:
```
/well/rescomp/eb/software/Boost/1.66.0-intel-2017b/lib/libboost_thread.a(thread.o): In function `boost::this_thread::hidden::sleep_for(timespec const&)':
thread.cpp:(.text+0x461c): undefined reference to `clock_gettime'
thread.cpp:(.text+0x469b): undefined reference to `clock_gettime'
/well/rescomp/eb/software/Boost/1.66.0-intel-2017b/lib/libboost_thread.a(thread.o): In function `boost::this_thread::no_interruption_point::hidden::sleep_until_realtime(timespec const&)':
thread.cpp:(.text+0x6af0): undefined reference to `clock_gettime'
thread.cpp:(.text+0x6ba7): undefined reference to `clock_gettime'
make[2]: *** [src/express] Error 1
make[2]: Leaving directory `/dev/shm/eXpress/1.5.1/intel-2017b/express-1.5.1-src'
make[1]: *** [src/CMakeFiles/express.dir/all] Error 2
make[1]: Leaving directory `/dev/shm/eXpress/1.5.1/intel-2017b/express-1.5.1-src'
make: *** [all] Error 2
```

**Comment by boegel** [comment]
BamTools 2.5.1 fleshed out to #5736 (so it can be used as a dep for #5734)

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 1 (3 easyconfigs in this PR)
rescomp2 - Linux centos 6.8, Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz, Python 2.7.11
See https://gist.github.com/fc1036a4e99c10eafb2bc39bc394f065 for a full test report.

**Comment by verdurin** [comment]
Same failure again.

**Comment by boegel** [comment]
@verdurin Haven't found the time to dive into this issue...

Looks like an incompatibility with the included Boost?

This probably worked for me because we have Boost headers available in the OS :(

**Comment by boegel** [comment]
@verdurin I can't reproduce the problem you were seeing on a CentOS 7.6 system where `Boost` is not installed on the system. Is this issue specific to CentOS 6.x?

**Comment by boegel** [comment]
Looks like it's a missing `-lrt` somewhere, see also https://svn.boost.org/trac10/ticket/8448 for example

**Comment by boegel** [comment]
@verdurin I was able to reproduce the issue you were seeing on a CentOS 6.x system, adding `-lrt` fixes the problem for me.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2001.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/e14444a953cb63b923670e3d5612f01b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (2 easyconfigs in this PR)
node2447.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/6062b6f272e0f0b8b62f053ee597d5be for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3100.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/feaff9ef8275012076fcaec7a00704b4 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (2 easyconfigs in this PR)
rescomp2 - Linux centos linux 7.3.1611, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/ec20884689920c958855dda2dc2a52b4 for a full test report.

File: easybuild/easyconfigs/e/eXpress/eXpress-1.5.1-intel-2017b.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+easyblock = 'CMakeMake'
```
Comment (verdurin) [inline easybuild/easyconfigs/e/eXpress/eXpress-1.5.1-intel-2017b.eb]: @boegel please restore the original attribution

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**